test/watcher.js

```diff
@@ -431,7 +447,8 @@ group('chokidar is installed', function (beforeEach, test, group) {
 		t.plan(2);
 
 		files = ['foo-{bar,baz}.js'];
-		api.excludePatterns = ['!*bar*'];
+		// TODO(@jamestalmage, @novemberborn): There is no way for users to actually set exclude patterns yet.
+		avaFiles.defaultExcludePatterns.returns(['!*bar*']);
```

Comment:
@novemberborn - Currently, there is no way for users to set the exclude patterns. Is this a feature that needs to be exposed? 

You access `api.excludePatterns` in watcher, but that's a static value that never changes (I guess an API consumer could, but we haven't published that, so ...).

Should we open an issue about this, or is `avaFiles.defaultExcludePatterns()` all you need.